user-interface/src/main/java/life/qbic/datamanager/views/general/DialogWindow.java

```diff
@@ -29,6 +33,13 @@ protected DialogWindow() {
     confirmButton.addClickListener(this::onConfirmClicked);
   }
 
+  protected void initCancelShortcuts(Command onCreationCanceled) {
```

Comment:
NIT: Since you're changing the cancel shortcut you're theoretically not initializing them 🤓 
```suggestion
  protected void specifyCancelShortcuts(Command onCreationCanceled) {
```